lib/schedule/minischedule/load.ex

```diff
@@ -119,9 +119,9 @@ defmodule Schedule.Minischedule.Load do
 
   @spec trip_in_operator(Activity.t(), Trip.t()) :: boolean()
   defp trip_in_operator(%Activity{activity_type: "Operator"} = activity, trip) do
-    trip.start_time >= activity.start_time and
-      trip.end_time <= activity.end_time and
-      String.contains?(trip.block_id, activity.partial_block_id)
+    String.contains?(trip.block_id, activity.partial_block_id) and
```

Comment:
this is something I noticed before and meant to ask about, what's the deal with sometimes having partial and sometimes what I assume are complete block IDs?

Reply:
`trips.csv` lists its block ids like `"A57-11"`, which is what GTFS uses, but `activities.csv` only gives us the `"57-11"` part. So we have to do this `String.contains?` matching instead of `==`.

An alternative might be to try to reconstruct the full block id. The missing letter, which represents the garage, shows up (in lowercase) as the 3rd character of the schedule_id, but using the trip's full block id seemed easier.

Reply:
No doubt, just wondering what that was about.